source/reference/auth-plugin-jdbc.txt

```diff
@@ -35,10 +37,9 @@ To download the plugin directly:
 
 To use Maven:
 
-1. Include the authentication plugin module library in the ``CLASSPATH``. The
-   Maven coordinates are:
+1. Include the authentication plugin module library in the
+   ``CLASSPATH``. The Maven coordinates are:
```

Comment:
make the structure of these steps parallel. One says: `The Maven coordinates are`; the other says: `Its Maven coordinates are`.